From a3c596d3609924400d30e7e9fd23d7d94285b078 Mon Sep 17 00:00:00 2001
From: "Dr. Greg Wettstein" <greg@enjellic.com>
Date: Sat, 11 Mar 2017 08:43:13 -0600
Subject: [PATCH 12/24] Change name of structure containing actor identity
 elements.

This update is a straight forward and idempotent change in
functionality.

The identity elements of the actor process was storage in a
structure named 'identity_elements'.  When subject for subject
identity elements was added the structure name used was
'subject_identity'.  In order to maintain consistency the
'identity_elements' structure was renamed to 'actor_identity' with
corresponding changes to all references to the structure.
---
 security/integrity/ima/ima_identity.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/security/integrity/ima/ima_identity.c b/security/integrity/ima/ima_identity.c
index 4fd90d8121de..234e61c364a8 100644
--- a/security/integrity/ima/ima_identity.c
+++ b/security/integrity/ima/ima_identity.c
@@ -21,7 +21,7 @@
 #include "ima.h"
 
 /* Structure defining actor identity elements. */
-struct identity_elements {
+struct actor_identity {
 	uid_t uid;
 	uid_t euid;
 	uid_t suid;
@@ -85,7 +85,7 @@ struct forensic_event {
 	int function;
 	char *process;
 	char *pathname;
-	struct identity_elements identity;
+	struct actor_identity identity;
 };
 
 static DEFINE_MUTEX(forensic_event_list_mutex);
@@ -118,7 +118,7 @@ static atomic_t map_opencount = ATOMIC_INIT(1);
  *	elements used to define an actor's identity state.
  */
 
-static void get_identity(struct identity_elements *id)
+static void get_identity(struct actor_identity *id)
 
 {
 	kernel_cap_t eff, per, inh;
@@ -778,7 +778,7 @@ static int ima_get_actor_identity(struct crypto_shash *tfm, char *mapping)
 		struct shash_desc shash;
 		char ctx[crypto_shash_descsize(tfm)];
 	} desc;
-	struct identity_elements identity;
+	struct actor_identity identity;
 
 	desc.shash.tfm = tfm;
 	desc.shash.flags = 0;
-- 
2.11.0

